AWS_CRUD_USER/userUpdate.js

```diff
@@ -0,0 +1,63 @@
+const AWS = require('aws-sdk');
+const ddb = new AWS.DynamoDB.DocumentClient();
+
+exports.handler = async (event) => {
+    console.log(event);
+    
+        
+    var bodyJSON = JSON.parse(event.body);
+    
+    var userId = bodyJSON.userId || null;
+    var name = bodyJSON.name|| null;
+    var lastName = bodyJSON.lastName || null;
+    var userName = bodyJSON.userName || null;
+    var password = bodyJSON.password || null;
+    var email = bodyJSON.email || null ;
+    var avatar = bodyJSON.avatar || null ;
+    var city = bodyJSON.city || null ;
+    var language = bodyJSON.language || null ;
+    var gameCollection = bodyJSON.gameCollection || null ;
```

Comment:
Remember to replace this initialization with only the keys provided in the received `event.body`